src/cli.rs

```diff
@@ -51,6 +51,10 @@ pub enum SubCommands {
         /// The mod loader will not be checked for this mod
         #[clap(long, short = 'M', alias = "dont-check-mod-loader")]
         ignore_mod_loader: bool,
+
+        /// Ignore the donation message
+        #[clap(long, short = 'H', alias = "hide-donation-message")]
```

Comment:
The alias here is unnecessary, clap will automatically convert Rust snake_case variables and PascalCase subcommands to the standard kebab-case used in CLIs.

```diff
- #[clap(long, short = 'H', alias = "hide-donation-message")]
+ #[clap(long, short = 'H')]
```

Reply:
Also I think `Hide the donation message` is good for the description too